### PR TITLE
Simplify single_keyword

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19938)

### CODE REVIEW

File: components/style/properties/longhand/inherited_text.mako.rs [hunk 1]
```diff
@@ -60,16 +60,18 @@ ${helpers.single_keyword("word-break",
...
-                                  values="auto none inter-word"
-                                  extra_gecko_values="inter-character"
-                                  extra_specified="${'distribute' if product == 'gecko' else ''}"
-                                  gecko_enum_prefix="StyleTextJustify"
-                                  animation_value_type="discrete"
-                                  gecko_pref="layout.css.text-justify.enabled"
-                                  flags="APPLIES_TO_PLACEHOLDER",
-                                  spec="https://drafts.csswg.org/css-text/#propdef-text-justify">
-
+<%helpers:single_keyword
```
Comment (emilio) [inline components/style/properties/longhand/inherited_text.mako.rs]: Can we change this to the:

```
${helpers.single_keyword(...)}
```

syntax while you're here? I think we use it more often.

(followup / E-easy is fine too if you prefer).

File: components/style/properties/longhand/box.mako.rs [hunk 2]
```diff
@@ -105,18 +107,20 @@ ${helpers.single_keyword("position", "static absolute relative fixed sticky",
...
+    name="clear"
+    values="none left right both"
+    // https://drafts.csswg.org/css-logical-props/#float-clear
+    extra_specified="inline-start inline-end"
+    needs_conversion="True"
+    gecko_inexhaustive="True"
+    animation_value_type="discrete"
+    gecko_enum_prefix="StyleClear"
+    gecko_ffi_name="mBreakType"
+    spec="https://www.w3.org/TR/CSS2/visuren.html#flow-control"
```
Comment (emilio) [inline components/style/properties/longhand/box.mako.rs]: If while you're here you change this to https://drafts.csswg.org/css-box/#propdef-clear it'll make me very happy :)

**Review by emilio — APPROVED** [review]
r=me, nice one :)

File: components/style/properties/longhand/inherited_text.mako.rs (hunk 1 above)
Comment (nox) [reply components/style/properties/longhand/inherited_text.mako.rs]: Can't do that, given I need to implement `ToComputedValue` by hand for the Gecko product.

**Pull request merged**